packages/df-blogs/src/index.tsx

```diff
@@ -51,8 +53,11 @@ class App extends React.PureComponent<Props> {
           <Route path={`${basePath}/new`} component={EditBlog} />
           <Route path={`${basePath}/posts/:id/edit`} component={EditPost} />
           <Route path={`${basePath}/posts/:id`} component={ViewPostById} />
+          <Route path={`${basePath}/comments/:id/edit`} component={EditComment} />
+          <Route path={`${basePath}/comments/:id`} component={ViewCommentById} />
           <Route path={`${basePath}/:id/edit`} component={EditBlog} />
           <Route path={`${basePath}/:id/newPost`} component={NewPost} />
+          <Route path={`${basePath}/:id/newComment`} component={NewComment} />
```

Comment:
Comment should be created under a post page:
```suggestion
          <Route path={`${basePath}/posts/:id/newComment`} component={NewComment} />
```

Reply:
Also this Route should be located just below this route:
```<Route path={`${basePath}/posts/:id/edit`} component={EditPost} />```

Reply:
There should be the same in the post creation link. Also, I think it will be better if there will be blog id at comment creation link and comments links.